### PR TITLE
ci: fix error check in version upload

### PR DESCRIPTION
**Pull request by joemfb**
It looks like this has been broken since the repository split. The version upload works, but the error check does not.

### CODE REVIEW

**Comment by matthew-levan** [comment]
Yes I've known about this for a long time, thanks for fixing it. Approved. Let's wait until passing CI to merge just in case.

**Review by matthew-levan — APPROVED** [review]

**Review by barter-simsum — APPROVED** [review]
IA

**Comment by joemfb** [comment]
It won't run in this PR build, only in the push to `develop`.

**Comment by joemfb** [comment]
worked: https://github.com/urbit/vere/actions/runs/5425664779/jobs/9867110484#step:5:58

**Pull request merged**
